apps/snitch_core/priv/repo/migrations/20180213103123_add_money_with_currency_type_to_postgres.exs

```diff
@@ -3,11 +3,11 @@ defmodule Core.Repo.Migrations.AddMoneyWithCurrencyTypeToPostgres do
 
   def up do
     execute """
-      CREATE TYPE public.money_with_currency AS (currency_code char(3), amount numeric(20,8))
+      CREATE TYPE #{ prefix() || "public" }.money_with_currency AS (currency_code char(3), amount numeric(20,8))
```

Comment:
@ramansah How can I run this on systems already with previous migration in DB.
We never update a migration, instead we create another migration to update the DB.